siftseek/endpoints/seeker/apply/seeker_apply_post.py

```diff
@@ -0,0 +1,42 @@
+from flask import Response, jsonify, request
+from marshmallow import ValidationError
+
+from siftseek.models.db import db
+from siftseek.endpoints.seeker import seeker
+from siftseek.schemas.job_application_schema import application_schema
+
+
+@seeker.post("/apply/<int:seeker_id>")
+def apply_for_job(seeker_id: int) -> tuple[Response, int]:
```

Comment:
a little confused. I think you may need some additional endpoints. `apply_for_job` sounds like you're submitting an application which i think is what this route is doing. But don't you need to match a job application to a job posting?

Reply:
I haven't matched to the job posting yet because I haven't created the job posting model, and I didn't want tests to throw errors just because of that. I will circle back and add that functionality there when I create the company endpoint.